### PR TITLE
Changing font name/size for ComboBoxItem

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): #2065

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
Font name/size, used in ComboBoxItems, are smaller than used for other UI elements, e.g. ComboBox content (selected item).

## What is the new behavior?
The same font is used for ComboBoxItems.

## PR Checklist

Please check if your PR fulfills the following requirements:
- [X] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [X] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [X] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by davidjohnoliver** [comment]
Can you confirm that you copied this from the UWP default style?

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml [hunk 1]
```diff
@@ -791,7 +791,12 @@
 				Value="{ThemeResource SystemControlForegroundBaseHighBrush}" />
 		<Setter Property="UseSystemFocusVisuals"
 				Value="True" />
-		<Setter Property="Template">
+		<Setter Property="FontFamily"
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml]: Did you pick this from the UWP `generic.xaml` file ?

Comment (pkar70) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml]: Yes, it is copied from ComboBox template, from DefaultComboBoxStyle, lines 514-517.
I think it was only omission (i.e., setting fontsize & fontfamily was missing in ComboBoxItem).

**Pull request merged**